converters/toZigbee.js

```diff
@@ -509,7 +511,7 @@ const converters = {
                         // 0x09 Sleep
                         attrId: zclId.attr(cid, attrId).value,
                         dataType: zclId.attrType(cid, attrId).value,
-                        attrData: value,
+                        attrData: utils.getKeyByValue(common.thermostat_system_modes, value),
```

Comment:
Could you set the fallback to the value for both cases? In this way it is backwards compatible.

Reply:
Great idea, I didn't even think about that.